client/src/components/SelectionContainer.tsx

```diff
@@ -140,12 +165,14 @@ export default function SelectionContainer({
         </Box>
         {/* only render link tag if selection criteria are met */}
         {!isSelected || !selectedEqual ? (
-          <CustomButton
-            variant="contained"
-            disabled={!isSelected || !selectedEqual}
-          >
-            Assign
-          </CustomButton>
+          <div className={classNames(classes.buttonContainer)}>
+            <CustomButton
```

Comment:
this button is not the same as on the assignments page. Is it possible to match it?

Reply:
I was able to match it by editing `<CustomButton  />` and using inline styling instead. Not sure why the useStyles() weren't working correctly in this button.